massa-consensus-worker/src/worker/init.rs

```diff
@@ -41,7 +41,10 @@ pub fn create_genesis_block(
 ) -> Result<SecureShareBlock, ConsensusError> {
     let keypair = &cfg.genesis_key;
     let header = BlockHeader::new_verifiable(
+        // NEW TODO: what to implement here in case of restart?
```

Comment:
And more important question, what happen for a MipStore transitionning during a network shutdown?

Reply:
Added to #3631